data/brands/amenity/cinema.json

```diff
@@ -144,14 +144,15 @@
       "displayName": "Cinema City",
       "id": "cinemacity-26c608",
       "locationSet": {
-        "include": ["cz", "hu", "pl", "ro", "sk"]
+        "include": ["bg, "cz", "hu", "pl", "ro", "sk"]
       },
       "tags": {
         "amenity": "cinema",
         "brand": "Cinema City",
         "brand:wikidata": "Q543651",
         "brand:wikipedia": "en:Cinema City International",
         "name": "Cinema City"
+        "operator": "Cineworld"
```

Comment:
Their Wikipedia article suggests that Cineworld is just the parent company and that Cinema City operates the cinemas
```suggestion
```